docs/src/components/RefinementList.md

```diff
@@ -25,6 +25,7 @@ Name | Type | Default | Description | Required
 ---|---|---|---|---
 attribute | string | | The attribute to refine on click | yes
 searchable | boolean | `false` | You can also search within the options of this | no
+placeholder | string | "Search here …" | If searchable is `true`, this will be the placeholder for that search box | no
```

Comment:
Placeholder is a bit broad in the context of a RefinementList. Why not `searchablePlaceholder` ?

Reply:
We indeed call this `searchablePlaceholder` in InstantSearch.js 3 ([see new options](https://github.com/algolia/instantsearch.js/blob/838abecceacad5bb72587697eb9c64642c325882/src/widgets/refinement-list/refinement-list.js#L130)).